src/fpm_command_line.f90

```diff
@@ -132,6 +134,19 @@ module fpm_command_line
     logical                       :: registry_cache = .false.
 end type
 
+type, extends(fpm_cmd_settings)   :: fpm_search_settings
```

Comment:
It would be good to add some comments describing the contents of each variable here (for example, like you added in the --help section)

Reply:
sure, I will add them.